osmtm/views/project.py

```diff
@@ -12,6 +12,7 @@
     TaskState,
     TaskLock,
     License,
+    INVALIDATED,
```

Comment:
Not importing constants directly and using properties of TaskState instead: https://github.com/hotosm/osm-tasking-manager2/commit/d8c8ce608719206a39cc8c6db3246322ed7454e6